core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -219,7 +219,8 @@ object Boot extends IOApp with LazyLogging {
         workbenchMetricBaseName = metricsPrefix
       )
 
-      val importServiceDAO = new HttpImportServiceDAO(conf.getString("avroUpsertMonitor.server"))
+      val importServiceDAO =
+        new HttpImportServiceDAO(conf.getString("avroUpsertMonitor.server"), conf.getString("avroUpsertMonitor.cwds"))
```

Comment:
this new config value was added in https://github.com/broadinstitute/terra-helmfile/pull/5233